src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -298,10 +298,10 @@ async function initializeData(): Promise<void> {
 function selectDataPlaneOverview(name: string | null): void {
   if (name && props.dataPlaneOverviews.length > 0) {
     dataPlaneOverview.value = props.dataPlaneOverviews.find((data) => data.name === name) ?? props.dataPlaneOverviews[0]
-    patchQueryParam('name', dataPlaneOverview.value.name)
+    patchQueryParam('dpp', dataPlaneOverview.value.name)
```

Comment:
Kinda off topic question: Do we still need `patchQueryParam` now we've moved to history API routing?

Reply:
Yes and the reason for this is that using the router’s push method will cause a navigation when we don’t want that. We _only_ want to change the URL. We don’t want a navigation to happen.